perception/autoware_traffic_light_visualization/src/traffic_light_roi_visualizer/shape_draw.cpp

```diff
@@ -42,14 +46,13 @@ void drawShape(
   const int fill_rect_h = std::max(shape_img.rows, text_size.height) + 10;
 
   const cv::Point rect_position(position.x, position.y - fill_rect_h);
-
   if (
     rect_position.x < 0 || rect_position.y < 0 || rect_position.x + fill_rect_w > image.cols ||
     position.y > image.rows) {
     // TODO(KhalilSelyan): This error message may flood the terminal logs, so commented out
     // temporarily. Need to consider a better way.
 
-    // std::cerr << "Adjusted position is out of image bounds." << std::endl;
+    std::cerr << "Adjusted position is out of image bounds." << std::endl;
```

Comment:
It seems better to use ROS_WARN instead of std::cerr.

Reply:
@yukkysaito 
This source code does not depend on ROS. So using `cerr` is better than `ROS_WARN` adding ros header, I think. What do you think?